src/archivist/index.js

```diff
@@ -22,7 +22,10 @@ export const AVAILABLE_EVENTS = [
   'versionRecorded',
   'firstVersionRecorded',
   'versionNotChanged',
-  'recordsPublished',
+  'startRefilterRun',
+  'refilterRunCompleted',
```

Comment:
At first sight it looked weird to me to not have a `start` and `end`
I understand you've thought this through but i'd like to undestand why you picked this instead of
`refilterRunStart` or `refilterRunStarted` (to keep the verb at the end)

Reply:
You're right. It's better with `refilterRunStarted`.